src/net/RTP/RTPChannel.cs

```diff
@@ -362,7 +362,7 @@ public enum RTPChannelSocketsEnum
     /// </summary>
     public class RTPChannel : IDisposable
     {
-        private static readonly bool isMono = Type.GetType("Mono.Runtime") != null;
+        internal static readonly bool isMono = Type.GetType("Mono.Runtime") != null;

```

Comment:
This is not particular to `RTPChannel` or its users. It should be moved to its own type and made public.

Its value can also be cached:

```suggestion
        public static readonly bool IsMono { get; } = Type.GetType("Mono.Runtime") != null;
```